### ISSUE TITLE
ls-gan loss

### ISSUE THREAD

**Issue by PRigas96** [junyanz/pytorch-CycleGAN-and-pix2pix#1548]
hello i was searching through the code for the lsgan loss and found:
`if gan_mode == 'lsgan':
            self.loss = nn.MSELoss()`
`if self.gan_mode in ['lsgan', 'vanilla']:
            target_tensor = self.get_target_tensor(prediction, target_is_real)
            loss = self.loss(prediction, target_tensor)`

i would be pleased if you could explain how is this equal to:
![image](https://user-images.githubusercontent.com/93339610/223080042-9ff7a7b4-298d-42fc-ac99-df8d73b6aaa7.png)

for example, where are a,b and c valued.

**Comment by junyanz** [comment]
b=1, a=0, and c=1. We use [MSELoss](https://pytorch.org/docs/stable/generated/torch.nn.MSELoss.html) to implement L2 loss.
